### PR TITLE
Remove unnecessary role="img"

### PR DESCRIPTION
**Pull request by hannahiss**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Related issues

#2667 

### Description

Remove unnecessary role="img" on aria-hidden SVGs or `` 

### Motivation & Context

This issue has been seen while fixing a11y audit: 
- aria-hidden images should not have a `role`
- tags `` are used for images only and should not have a `role="img"`

### Types of change

- Bug fix (non-breaking which fixes an issue)

### Live previews

- <https://deploy-preview-2704--boosted.netlify.app/docs/about/brand/>
- <https://deploy-preview-2704--boosted.netlify.app/docs/components/scrollspy/>
- <https://deploy-preview-2704--boosted.netlify.app/docs/customize/color-palette/>
- <https://deploy-preview-2704--boosted.netlify.app/docs/dark-mode/>
- <https://deploy-preview-2704--boosted.netlify.app/docs/examples/download-app/>

### CODE REVIEW

File: site/content/docs/5.3/dark-mode.md
Comment (louismaximepiton) [inline site/content/docs/5.3/dark-mode.md:1]: 3 are missing in here

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
1 is missing also in both `orange-global-headers.html` and `orange-supra.html`.

**Review by louismaximepiton — APPROVED** [review]
LGTM 🚀

**Review by julien-deramond — COMMENTED** [review]
The HTML markup changed (slightly) for the headers, so it might be useful to mention it in the migration guide, isn't it?

(Note: there are some conflicts to resolve in the PR due to other PRs being merged)

File: site/content/docs/5.3/migration.md [hunk 1]
```diff
@@ -23,10 +23,13 @@ toc: true
 #### Buttons
   - <span class="badge text-bg-success">New</span> X social button.
 
+#### Orange navbar
```
Comment (hannahiss) [inline site/content/docs/5.3/migration.md:26]: I'm going crazy with all these PRs... This was already done in #2690 [here](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2690/files#diff-ff554e03e75f214021058bea325f4efdbe435154b706ecf220c9b29653a4ba8b), and in a better way I find... What do you think @julien-deramond

Comment (hannahiss) [reply site/content/docs/5.3/migration.md]: Anyway, I made some changes: 
- fix only `role="img"` in this PR #2704 => Change migration guide accordingly
- fix only alt attribute in PR #2690 => Change migration guide accordingly

**Comment by julien-deramond** [comment]
This branch has conflicts that must be resolved after the recent merges in the _main_ branch

**Comment by julien-deramond** [comment]
Apparently, thanks to your PR, we can remove one rule from `vnu-jar.mjs` → https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2704/files#diff-4ccadc344c026b116648a6acac72d1d643ec8508cc92d477371734a13c0e7e80

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**